### PR TITLE
chore: add cyberdevv

### PR DESCRIPTION
**Pull request by CyberDevv**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/914"></a>

### CODE REVIEW

File: public/data/CyberDevv.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Odesola Ibrahim",
+  "type": "personal",
+  "bio": "I am a Full-Stack Developer using React and Node.js",
+  "avatar": "https://github.com/cyberdevv.png",
```
Comment (Panquesito7) [inline public/data/CyberDevv.json:5]:
```suggestion
  "avatar": "https://github.com/CyberDevv.png",
```

File: public/data/CyberDevv.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/cyberdevv",
```
Comment (Panquesito7) [inline public/data/CyberDevv.json:9]:
```suggestion
      "url": "https://github.com/CyberDevv",
```

File: public/data/CyberDevv.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      "icon": "github"
+    },
+
+    {
+      "name": "Connect with me on Linkedin",
```
Comment (Panquesito7) [inline public/data/CyberDevv.json:14]:
```suggestion
      "name": "Connect with me on LinkedIn",
```

File: public/data/CyberDevv.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      "url": "https://www.linkedin.com/in/odesolaibrahim/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/cyberdevv",
+      "icon": "twitter"
+    },
+    {
+      "name": "Checkout my portfolio",
```
Comment (Panquesito7) [inline public/data/CyberDevv.json:24]:
```suggestion
      "name": "Check out my portfolio",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/CyberDevv.json (hunk 1 above)
Comment (nhcarrigan) [reply public/data/CyberDevv.json:5]: Pretty sure URLs are case-insensitive.

Comment (Panquesito7) [reply public/data/CyberDevv.json:5]: The current link should work, but I think it's best to use its correct syntax.

File: public/data/CyberDevv.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+      "url": "https://ibrahim-odesola.netlify.app/",
```
Comment (krishguptadev) [inline public/data/CyberDevv.json:25]:
```suggestion
      "url": "https://ibrahim-odesola.netlify.app",
```

Comment (Panquesito7) [reply public/data/CyberDevv.json:25]: This doesn't matter at all, IMO. It can be kept. 🙂

**Review by emmalearnscode — APPROVED** [review]

File: public/data/CyberDevv.json (hunk 1 above)
Comment (emmalearnscode) [reply public/data/CyberDevv.json:5]: @Panquesito7 This is something which has no effect whatsoever and achieves nothing by changing it.

**Comment by emmalearnscode** [comment]
Thanks for your contribution!

**Pull request merged**